OuterWildsRandomSpeedrun/SpawnPointPool.cs

```diff
@@ -5,10 +5,29 @@
 
 namespace OuterWildsRandomSpeedrun
 {
+    public enum Area
+    {
+        none,
+        sun_station,
```

Comment:
code LGTM, but could we use PascalCasing for these? That seems to be the idiom elsewhere